### PR TITLE
Minor build fixes

### PR DESCRIPTION
**Pull request by etienne-lms**
This P-R propose few minor implementation fixes toward strict types use and prototypes declaration.

### CODE REVIEW

File: bl1/aarch32/bl1_context_mgmt.c [hunk 1]
```diff
@@ -50,7 +50,7 @@ void *cm_get_context(uint32_t security_state)
 	return &bl1_cpu_context[security_state];
 }
 
-void cm_set_next_context(void *cpu_context)
+static void cm_set_next_context(void *cpu_context)
```
Comment (soby-mathew) [inline bl1/aarch32/bl1_context_mgmt.c]: I think the prototype was missed out and it is better to add a prototype in context_mgmt.h.

Comment (etienne-lms) [inline bl1/aarch32/bl1_context_mgmt.c]: True, indeed for the bl1_fwu. I'll fix.

**Comment by etienne-lms** [comment]
Fix from comment: replace faulty commit "context_mgmt: declare cm_set_next_context() as static for aarch32" with commit "context_mgmt: declare extern cm_set_next_context() for AArch32".
Rebased (no update required).

**Comment by danh-arm** [comment]
jenkins: test this please

**Pull request merged**